Source/Android/TealiumUnity/unity/src/main/java/com/tealium/TealiumUnity.kt

```diff
@@ -7,6 +7,7 @@ import com.tealium.core.Tealium
 import com.tealium.core.consent.ConsentCategory
 import com.tealium.core.consent.ConsentStatus
 import com.tealium.remotecommanddispatcher.remoteCommands
+import com.tealium.remotecommands.BuildConfig
```

Comment:
Can remove, as you want to use the `BuildConfig.TAG` for this project, not the one for remote commands.